src/Automation/Automation/Cmdlet/UpdateManagement/NewAzureAutomationSoftwareUpdateConfiguration.cs

```diff
@@ -40,15 +42,39 @@ public class NewAzureAutomationSoftwareUpdateConfiguration : AzureAutomationBase
         [Parameter(ParameterSetName = AutomationCmdletParameterSets.Linux, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "Indicates that the software update configuration targeting Linux operating system machines.")]
         public SwitchParameter Linux { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "Indicates that the software update configuration will Only Reboot the machines.")]
```

Comment:
@jemex you can remove `ValueFromPipelineByPropertyName` from all `SwitchParameter` parameters